app/components/node/PoSSize.tsx

```diff
@@ -157,6 +180,20 @@ const PoSSize = ({
           bold
         />
       </Row>
+      <Row>
+        <Icon1 src={posSpace} />
+        <Text>Max file size - {formatBytes(maxFileSize)}: </Text>
+        <Tooltip width={200} text="Some text" />
```

Comment:
Replace "Some text" with "PoS data will be stored into a bunch of files with the specified max file size".
I think it is better for the beginning, than just a placeholder :)